packages/amplify-category-geo/src/commands/geo.ts

```diff
@@ -0,0 +1,43 @@
+import { category } from '../constants';
+
+module.exports = {
+  name: category,
+  run: async (context: any) => {
+    if (/^win/.test(process.platform)) {
```

Comment:
why is this only for windows? Also, for checking windows, use `process.platform.startsWith('win')`

Reply:
I am not 100% sure about this one. All the other CLI category plugins also have this check.